### ISSUE TITLE
Version 4.4 crashes on launch

### ISSUE THREAD

**Issue by huntermonk** [dji-sdk/Mobile-SDK-iOS#206]
#### Is this issue about bugs or crash issues of DJI iOS SDK and Sample Code?

Yep, then please delete this template and provide the following infos for us to help investigate the issue:

**-  Description of the issues.**
After upgrade to `4.4` from `4.3.2`, crashes on launch.

**-  Steps to reproduce the bug or crash issues(It would be great if you can provide this)**
Upgrade to `4.4` and launch.

**-  Crash logs (If you can find it, it would be very helpful)**
```
dyld: Symbol not found: _DJICameraParamDigitalFilter
  Referenced from: /private/var/containers/Bundle/Application/21CB2214-84D7-42BE-820A-8D47272A0E60/Teleport.app/Frameworks/DJIUILibrary.framework/DJIUILibrary
  Expected in: /private/var/containers/Bundle/Application/21CB2214-84D7-42BE-820A-8D47272A0E60/Teleport.app/Frameworks/DJISDK.framework/DJISDK
 in /private/var/containers/Bundle/Application/21CB2214-84D7-42BE-820A-8D47272A0E60/Teleport.app/Frameworks/DJIUILibrary.framework/DJIUILibrary
```

**-  DJI iOS SDK version you are using (Like iOS SDK 3.4, etc)**
`iOS SDK 4.4`

**-  DJI Product you are using (Like Phantom 4, Mavic Pro, etc)**
None.

**-  iOS system version you are using (Like iOS 8.3, iOS 9, etc)**
iOS 11.2.1

**-  iOS device type you are using (Like iPhone5s, iPhone 6, iPad Air Wifi,  iPad mini 3 LTE, etc)**
iPhone X

**-  Xcode version you are using (Like Xcode 7.0)**
Xcode 9.2

```
dyld: Symbol not found: _DJICameraParamDigitalFilter
  Referenced from: /private/var/containers/Bundle/Application/21CB2214-84D7-42BE-820A-8D47272A0E60/Teleport.app/Frameworks/DJIUILibrary.framework/DJIUILibrary
  Expected in: /private/var/containers/Bundle/Application/21CB2214-84D7-42BE-820A-8D47272A0E60/Teleport.app/Frameworks/DJISDK.framework/DJISDK
 in /private/var/containers/Bundle/Application/21CB2214-84D7-42BE-820A-8D47272A0E60/Teleport.app/Frameworks/DJIUILibrary.framework/DJIUILibrary
```

**Comment by huntermonk** [comment]
Sorry! As evident by the crash log, I just hadn't upgraded the UI library to `4.4`. Leaving this open just in case.

**Comment by oliverou** [comment]
Hi @huntermonk , so your issue is resolved by upgrading the UI Library to 4.4?

**Comment by huntermonk** [comment]
That is correct. Closing now.
